Generic Types, Traits and Lifetime/Generic Types, Traits and Lifetime/Generics in Vec/task-info.yaml

```diff
@@ -0,0 +1,12 @@
+type: edu
+files:
+- name: src/main.rs
+  visible: true
+  placeholders:
+  - offset: 43
+    length: 4
+    placeholder_text: // Something goes here
```

Comment:
An open placeholder with "//" affects all of the remaining line, leaving following stuff commented. I suggest using a "/**/" inline placeholder to avoid this